doc/nrf/device_guides/nrf70/nrf7002eb_54l15pdk_gs.rst

```diff
@@ -55,7 +55,7 @@ The pin mapping for the nRF54L15 PDK is as follows:
 +-----------------------------------+-------------------+-----------------------------------------------+
 
 .. note::
-   Also connect `VIO`=1.8V, `VBAT`=3.6V and `GND`.
+   Also connect ``VIO`` to 1.8V, ``VBAT`` to 3.6V and ``GND``.
```

Comment:
```suggestion
   Connect ``VIO`` to 1.8 V and ``VBAT`` to 3.6 V and ``GND``.
```

Reply:
Also, could you please change lines 28 an d29 as follows? I am unable to comment there.

"Pin mapping for nRF54L15 PDK
====================="